### ISSUE TITLE
Allow changing default Object colour

### ISSUE THREAD

**Issue by eishiya** [mapeditor/tiled#3861]
### Problem

The default grey Object colour can be hard to see when placed against some tiles, particularly ones with a lot of grey. This is especially an issue when editing and checking Tile Collisions.

Here, for example, it can be hard to tell whether the tile has had collision added or not, especially when you want to check quickly:
![image](https://github.com/mapeditor/tiled/assets/70545646/3e0b2e36-d695-49db-8a8a-08af769ee75b)

### Proposal
It would be nice if the default Object colour could be customised, similarly to how the grid colour can be changed.

### Existing alternatives
Currently, one can increase the Object line thickness, which can help sometimes, but can make the object cover up important artwork otherwise.
Another option is to use a custom class that sets a different colour. However, this gets needlessly tedious when you don't otherwise need to use a class, such as for most collisions.

(The visibility issue was reported by Lumin and the screenshot is also from them, I'm just posting this for them.)

**Comment by bjorn** [comment]
Right, I think this used to be possible by setting a color on a "custom class" with an empty name, but this no longer works (possibly since Tiled 1.9). Rather than re-introducing that behavior, it would indeed be better to add an explicit setting for this.

**Comment by MollahHamza** [comment]
I've implemented an explicit setting for the default object color. A "Default object color" picker has been added to the Preferences → Interface tab. The color defaults to gray (matching the previous hardcoded behavior) and updates all map objects live when changed.                                                                                             
                                                                                                                          
  
<img width="672" height="717" alt="Image" src="https://github.com/user-attachments/assets/cb999ebf-2ab0-4c9a-bc8d-737f2aa4e5ce" />

<img width="452" height="260" alt="Image" src="https://github.com/user-attachments/assets/0835518d-db2f-4fc7-af1e-529c835733a0" />

                                                                                                                          
  Does this look like the right approach?

**Comment by bjorn** [comment]
@MollahHamza Yes, that looks great! I'd only put the color setting alongside the other configurable colors. I've assigned you the issue, thanks! :-)

**Comment by MollahHamza** [comment]
Hi @bjorn, I've moved the color setting to sit alongside Grid color and Background fade color in the Interface group. PR incoming.
